### PR TITLE
Rework profile details

### PR DESCRIPTION
**Pull request by AlexandreSi**
Change My profile page to show the user's email and bio, introduced in #3123

Result:

image

### CODE REVIEW

**Comment by Silver-Streak** [comment]
SUPER minor comment, and while I know this isn't a huge deal, as an outsider looking in I'd really love to see these PRs with descriptions when submitted, so I can help connect the community on the discord/forums to any WIP upcoming changes to help reduce duplicate issues/fears/etc.

**Comment by 4ian** [comment]
Ah interesting remark - I think also it might be due to the PR being a draft but there should be no problem ensuring we add at least a quick description for PRs in the future :) 👍

**Comment by AlexandreSi** [comment]
Hi @Silver-Streak I didn't realize you were using the PRs as a source of info!
I will add a description to my PRs from now on, starting with this one!

**Comment by AlexandreSi** [comment]
@4ian I am not sure what to put instead of "No bio defined", something like "Edit your profile to tell us what you are using GDevelop for!" ?
Maybe I can add the same logic for the username. For now, there will be nothing next to the avatar if the user does not have a username

**Comment by 4ian** [comment]
> Maybe I can add the same logic for the username. For now, there will be nothing next to the avatar if the user does not have a username

Great idea!

As for the bio, yes you can go for a "No bio defined. Edit your profile to tell us what you are using GDevelop for!" :)

**Comment by AlexandreSi** [comment]
Final result:
image
image

File: newIDE/app/src/Profile/ProfileDetails.js [hunk 1]
```diff
@@ -17,25 +19,61 @@ type Props = {
...
+    <I18n>
+      {({ i18n }) => (
+        <Column>
+          <Line alignItems="center">
+            <Avatar src={getGravatarUrl(profile.email || '', { size: 40 })} />
+            <Spacer />
+            <Text
+              size="title"
+              style={{
+                color: profile.username ? 'black' : '#888',
```
Comment (4ian) [inline newIDE/app/src/Profile/ProfileDetails.js:31]: Risky as could not work well with a future theme (even if gray is fairly safe, black is not and won't show well on dark themes).
Instead, keep the default color and use something like `opacity: 0.8` to dim a bit the text.

File: newIDE/app/src/Profile/ProfileDetails.js [hunk 2]
```diff
@@ -17,25 +19,61 @@ type Props = {
...
+                )
+              }
+              readOnly
+              fullWidth
+              multiline
+              floatingLabelText={<Trans>Bio</Trans>}
+              floatingLabelFixed={true}
+              inputStyle={{
+                color: profile.description ? 'black' : '#888',
+              }}
```
Comment (4ian) [inline newIDE/app/src/Profile/ProfileDetails.js:62]: Here, prefer passing a hint text when the value is empty :) 
This way, no need for inputStyle.

File: newIDE/app/src/Profile/ProfileDetails.js (hunk 1 above)
Comment (AlexandreSi) [reply newIDE/app/src/Profile/ProfileDetails.js:31]: Oh yes, I forgot about the themes!

File: newIDE/app/src/Profile/ProfileDetails.js (hunk 2 above)
Comment (AlexandreSi) [reply newIDE/app/src/Profile/ProfileDetails.js:62]: I thought that there was a problem with hint text, but not at all indeed. Will use it

**Review by 4ian — APPROVED** [review]

**Pull request merged**